microprofile.ls/com.redhat.microprofile.ls/src/main/java/com/redhat/microprofile/services/MicroProfileCompletions.java

```diff
@@ -241,6 +221,58 @@ private static void collectPropertyKeySuggestions(int offset, Node node, Propert
 		}
 	}
 
+	/**
+	 * Collect profiles
+	 * @param offset            the offset where completion was invoked
+	 * @param key               the property key
+	 * @param model             the properties model
+	 * @param markdownSupported boolean determining if markdown is supported
+	 * @param list              the completion list
+	 */
+	private static void collectProfileSuggestions(int offset, PropertyKey key,
+			PropertiesModel model, boolean markdownSupported, CompletionList list) {
+
+		Range range = null;
+		Position currPosition = null;
+		String line = null;
+		TextDocument textDocument = model.getDocument();
+		try {
+			range = textDocument.lineRangeAt(offset);
+			currPosition = textDocument.positionAt(offset);
+			line = textDocument.lineText(currPosition.getLine());
+		} catch (BadLocationException e) {
+			LOGGER.log(Level.SEVERE, "In QuarkusCompletion#collectPropertyKeySuggestions, position error", e);

```

Comment:
Change `In QuarkusCompletion...` to `In MicroProfileCompletion...`. I notice that there are several comments which uses Quarkus in this class, please replace it with MicroProfile. (We should just having QuarkusModel in this class which uses Quarkus).